yolov5/helpers.py

```diff
@@ -65,11 +63,54 @@ def predict(self, image_list, size=640, augment=False):
         results = self.model(imgs=image_list, size=size, augment=augment)
         return results
 
+    def video_predict(self, video_path, view_img=True, img_size=640):
+        from yolov5.utils.datasets import LoadImages
+        from yolov5.utils.general import non_max_suppression
+        from utils.plots import Annotator, colors
+        import cv2
+
+        cap = cv2.VideoCapture(video_path)         
+        output = []
+        while True:
+            _, im = cap.read()
+            im0s = im.copy()
+            im = torch.from_numpy(im).permute(2, 0, 1).float().div(255.0).unsqueeze(0)
+            pred = self.model(im)
+            pred = non_max_suppression(pred, conf_thres=0.25, iou_thres=0.45, classes=None, agnostic=False)
```

Comment:
@kadirnar su sekilde parametreleri hardcoded yapmak yerine modelden cekebiliriz:

```suggestion
            pred = non_max_suppression(pred, conf_thres=self.model.conf, iou_thres=self.model.iou, classes=self.model.classes, agnostic=self.model.agnostic)
```